src/components/Login/Login.jsx

```diff
@@ -1,64 +1,101 @@
-  const [emailIsValid, setEmailIsValid] = useState();
-  const [enteredPassword, setEnteredPassword] = useState("");
-  const [passwordIsValid, setPasswordIsValid] = useState();
-  const [formIsValid, setFormIsValid] = useState(false);
 
-  // useEffect(() => {
-  //   const identifier = setTimeout(() => {
-  //     setFormIsValid(
-  //       enteredEmail.includes("@") && enteredPassword.trim().length > 6
-  //     );
-  //   }, 500);
+export default function Login (props) {
+  const navigate = useNavigate()
+  const [email,setEmail] = useState('')
+  const [password,setPassword] = useState('')
+    // const [enteredEmail, setEnteredEmail] = useState("");
+  // const [emailIsValid, setEmailIsValid] = useState();
+  // const [enteredPassword, setEnteredPassword] = useState("");
+  // const [passwordIsValid, setPasswordIsValid] = useState();
+  const [formIsValid, setFormIsValid] = useState(false);
+  const notify = () => toast.error('Неверный логин или пароль!', {
+    position: "top-center",
+    autoClose: 7000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
+    });
 
-  //   return () => {
-  //     clearTimeout(identifier);
-  //   };
-  // }, [enteredEmail, enteredPassword]);
+  // const emailChangeHandler = (event) => {
+  //   setEnteredEmail(event.target.value);
+  // };
 
-  const emailChangeHandler = (event) => {
-    setEnteredEmail(event.target.value);
-  };
+  // const passwordChangeHandler = (event) => {
+  //   setEnteredPassword(event.target.value);
+  // };
 
-  const passwordChangeHandler = (event) => {
-    setEnteredPassword(event.target.value);
-  };
+  // const validateEmailHandler = () => {
+  //   setEmailIsValid(enteredEmail.includes("@"));
+  // };
 
-  const validateEmailHandler = () => {
-    setEmailIsValid(enteredEmail.includes("@"));
-  };
-
-  const validatePasswordHandler = () => {
-    setPasswordIsValid(enteredPassword.trim().length > 6);
-  };
+  // const validatePasswordHandler = () => {
+  //   setPasswordIsValid(enteredPassword.trim().length > 6);
+  // };
 
   const submitHandler = (event) => {
     event.preventDefault();
-    props.onLogin(enteredEmail, enteredPassword);
+    props.onLogin(email, password)
   };
+  injectStyle();
+  const login = ()=> {
+
+    if(email==='' & password===''){
+       return
+    }else{
+        axios.post('http://62.113.99.92:8000/security/login',{
+            email: email,
+            password: password
+        })
+        .then(function(response){
+          console.log(response.data.token,'response.data.token')
+          if(response.data.token){
+            setToken(response.data.token)
+            setNewuser(response.data.name)
+            console.log(response.data)
+            window.location.reload()
+          }
+        })
+        .catch(function(error){
+          console.log(error,'error');
+          {notify()}
+        });
+    }
+}
 
   return (
```

Comment:
I think it's not good to have logic and jsx elements in one module. Much better will be split several modules